### PR TITLE
[MappingApplication] Minor clean up headers

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

This uses `#pragma once` and corrects tabs in header 

**🆕 Changelog**

- Uses `#pragma once` 
- Corrects tabs in header 
- Unify python namespace

### CODE REVIEW

File: applications/MappingApplication/custom_python/add_custom_utilities_to_python.h [hunk 1]
```diff
@@ -5,17 +5,16 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Philipp Bucher, Jordi Cotela
+//  Main authorse:    Philipp Bucher, Jordi Cotela
```
Comment (philbucher) [inline applications/MappingApplication/custom_python/add_custom_utilities_to_python.h:11]:
```suggestion
//  Main authors:    Philipp Bucher, Jordi Cotela
```

File: applications/MappingApplication/custom_modelers/mapping_geometries_modeler.h [hunk 2]
```diff
@@ -7,12 +7,10 @@
 //  License:         BSD License
 //                   Kratos default license: kratos/license.txt
 //
+//  Main authors:    Philipp Bucher
```
Comment (philbucher) [inline applications/MappingApplication/custom_modelers/mapping_geometries_modeler.h:10]: this one (header and source) was done by Peter, Tobi and me

**Review by philbucher — APPROVED** [review]
minor, otherwise thx

File: applications/MappingApplication/custom_python/add_custom_utilities_to_python.h (hunk 1 above)
Comment (loumalouomega) [reply applications/MappingApplication/custom_python/add_custom_utilities_to_python.h:11]: In a new PR

File: applications/MappingApplication/custom_modelers/mapping_geometries_modeler.h (hunk 2 above)
Comment (loumalouomega) [reply applications/MappingApplication/custom_modelers/mapping_geometries_modeler.h:10]: In a new PR

**Pull request merged**